src/lib/ui/Switch/Switch.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames'
+
+export interface SwitchProps {
+  checked: boolean
+  onChange: (checked: boolean) => void
+  disableLabel?: boolean
+}
+
+function Switch({
+  checked,
+  onChange,
+  disableLabel = true,
```

Comment:
Minor note, but default optional prop convention in this project is that missing is the same as falsy. This makes usage really clean since you can do `<Foo enableBar />` or `<Foo disableBar />` to opt in to the non-default Boolean behaivor.

So either the default should be to enable the label

```
  disableLabel = fasle,
```

or the prop name would change to match the default of opting in to show the label

```
  enableLabel = false,
```

Reply:
I'll go with enableLabel, good catch

Reply:
[05a1f27](https://github.com/seamapi/react/pull/377/commits/05a1f273fc62fd6c1b94970c139883877c6951ba)